index.md

```diff
@@ -7,7 +7,7 @@ StarCon is a two-day software engineering conference held at the **University of
 
 ## Who's speaking? Can I apply to speak?
 
-Check out our [invited keynotes and speakers](speakers/#keynotes)! Talk proposal details will be posted in September.
+Check out our [invited keynotes and speakers](speakers/#keynotes)! Learn about how you can speak at StarCon [here](/cfp/).
```

Comment:
This was in the commit I merged wasn't it? (Unless I got git and hg confused, I was working on both at the same time)